docs/v-slice/components/NoteStyles.md

```diff
@@ -0,0 +1,70 @@
+# Notestyles
+
+NoteStyles Change How The Notes Look
```

Comment:
A little to stub-y here. I would have excepted more detailed description (especially since it's responsible for a little more than note's look)